File: Python/Array/Array Subset of another array.py
from collections import defaultdict
def isSubset( a1, a2, n, m):
    map = defaultdict(int)
    for i in range(n):
        map[a1[i]] = 1
    
    for j in range(m):
        if map[a2[j]] != 1:
            return "No"
            
    return "Yes"  

File: C++/Array/Array Subset of another array.cpp
#include<bits/stdc++.h>
string isSubset(int a1[], int a2[], int n, int m) {
    
    unordered_map<int, int> map;
    for(int i = 0; i < n; i++){
        map[a1[i]] = 1;
    }
    
    for(int j = 0; j < m; j++){
        if(map[a2[j]] != 1){
            return "No";
        }
    }
    
    return "Yes";
}
